website/docs/r/cluster.html.markdown

```diff
@@ -348,6 +348,13 @@ In addition to all arguments above, the following attributes are exported:
 * `mongo_uri` - Base connection string for the cluster. Atlas only displays this field after the cluster is operational, not while it builds the cluster.
 * `mongo_uri_updated` - Lists when the connection string was last updated. The connection string changes, for example, if you change a replica set to a sharded cluster.
 * `mongo_uri_with_options` - connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.
+* `connection_strings` - Set of connection strings that your applications use to connect to this cluster.
```

Comment:
For these can we update w/ more complete descriptions from https://docs.atlas.mongodb.com/reference/api/clusters-get-all/ connectionStrings.X - these can be really challenging for users to understand so more tends to be better.  If it's too much text can be better summarize each and then link to the API doc?  

It's fine if it's a little less info on the data sources but for here it should be a bit more descriptive imho.